app/dedicated-cloud-gateways/custom-dns.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Custom Domains"
+content_type: reference
+layout: reference
+description: "{{site.konnect_short_name}} integrates domain name management and configuration with [Dedicated Cloud Gateways](/dedicated-cloud-gateways/)."
+
+products:
+    - gateway
+faqs:
+  - q: What should I do if my custom domain fails to attach in {{site.konnect_short_name}}?
+    a: |
+      If your custom domain fails to attach, check whether your domain has a Certificate Authority Authorization (CAA) record that restricts certificate issuance. 
+      {{site.konnect_short_name}} uses Google Cloud Public CA (`pki.goog`) to provision SSL/TLS certificates. If the CAA record doesn’t include `pki.goog`, certificate issuance will fail.
+
+      To resolve the issue:
+      1. Run `dig CAA yourdomain.com +short` to check for existing CAA records.
+      2. If a record exists but doesn’t allow `pki.goog`, update it.
+         `yourdomain.com.    CAA    0 issue "pki.goog"`
+      3. Wait for DNS propagation and try attaching your domain again.
+
+      If no CAA record exists, no changes are needed. For more details, see the [Let's Encrypt CAA Guide](https://letsencrypt.org/docs/caa/).
+
+  - q: How do I configure a custom domain in {{site.konnect_short_name}}?
+    a: |
+      In {{site.konnect_short_name}}, go to [**Gateway Manager**](https://cloud.konghq.com/us/gateway-manager/), select a Control Plane, open the **Overview** dashboard, click **Connect**, and save the **Public Edge DNS** URL. 
+      Then, navigate to **Custom Domains**, click **New Custom Domain**, enter your domain name, and save the CNAME and Content values.
+
+  - q: What DNS records do I need to add at my domain registrar for a custom domain?
+    a: |
+      You need to create CNAME records using the values provided in {{site.konnect_short_name}}.
+      * `_acme-challenge.example.com` → `_acme-challenge.9e454bcfec.acme.gateways.konghq.com`
+      * `example.com` → `9e454bcfec.gateways.konghq.com`
```

Comment:
This is already in the body of the page, do we need it here?

Reply:
I had it in there for search but you are right its just duplicate